packages/esify/lib/config.js

```diff
@@ -35,9 +35,12 @@ module.exports = function loadConfig() {
         NProgress: 'NProgress',
         FastClick: 'shopify-fastclick',
         Clipboard: 'clipboard',
+        Twine: 'twine',
+        Bindings: 'twine',
       },
       renameIdentifiers: {
         jQuery: '$',
+        Bindings: 'Twine',
```

Comment:
@Shopify/tnt are we cool with moving back to `Twine` as the main name for accessing Twine (over `Bindings`)?